packages/block-directory/src/components/downloadable-block-header/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BlockIcon } from '@wordpress/block-editor';
+import BlockRatings from '../block-ratings';
+
+function DownloadableBlockHeader( { icon, title, rating, ratingCount, onClick } ) {
+	return (
+		<div className="block-directory-downloadable-block-header__row">
+			{
+				icon.match( /\.(jpeg|jpg|gif|png)/ ) !== null ?
```

Comment:
Should we check that the extension is at the end of the string?

```suggestion
				icon.match( /\.(jpeg|jpg|gif|png)$/ ) !== null ?
```